roles/parse-docker-files/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Get a list of directory names in project
+  raw: ls ../
```

Comment:
that would still give me the contents of `project/ansible` instead of `project` which is what I want, would it not?